### DISCUSSION TITLE
Ability to search photos by the country they were clicked at

### DISCUSSION THREAD

**Discussion by vishnukvmd** [ente-io/ente#890] [Enhancements]
This will require geo-fencing data to accurately map co-ordinates to countries.

If there are public datasets available, please do share.

**Comment by ajsteele** [comment]
[Aves](https://github.com/deckerst/aves) 'uses a TopoJSON file from [World Atlas](https://github.com/topojson/world-atlas) under ISC licence', if that would be suitable. :)
